### ISSUE TITLE
README has invalid Maven badge for 2.12

### ISSUE THREAD

**Issue by SethTisue** [scala/scala-xml#149]
<img width="231" alt="screen shot 2017-05-10 at 10 30 44 am" src="https://cloud.githubusercontent.com/assets/161079/25912351/c78f15aa-356b-11e7-8e15-71bf1338aa22.png">

**Comment by ashawley** [comment]
Looks like there is an errant asterisk which shouldn't be there.

```
https://img.shields.io/maven-central/v/org.scala-lang.modules/scala-xml_2.12*.svg?
label=latest%20release%20for%202.12
```

![latest release for 2.12](https://img.shields.io/maven-central/v/org.scala-lang.modules/scala-xml_2.12.svg?label=latest%20release%20for%202.12)

It'd be nice to switch to Github markdown image notation instead of `<img/>` markup.

**Comment by SethTisue** [comment]
Off-topic: I'm at a Scala meetup at Prezi and (it came out in response to an audience question) they are using scala-xml for their XML processing. Yay scala-xml! I get focused on bugs sometimes and forget that it actually works fine for lots of people :-)

**Comment by ashawley** [comment]
We could even maintain a badge for 2.13!

![latest release for 2.13.0-M1](https://img.shields.io/maven-central/v/org.scala-lang.modules/scala-xml_2.13.0-M1.svg?label=latest%20release%20for%202.13.0-M1)

**Comment by ashawley** [comment]
I might change the labels from "latest release for 2.11" to just "scala 2.11", unless anyone objects.

----
[![Travis](https://img.shields.io/travis/scala/scala-xml.svg)](https://travis-ci.org/scala/scala-xml) [![latest release for 2.11](https://img.shields.io/maven-central/v/org.scala-lang.modules/scala-xml_2.11.svg?label=scala+2.11)](http://mvnrepository.com/artifact/org.scala-lang.modules/scala-xml_2.11) [![latest release for 2.12](https://img.shields.io/maven-central/v/org.scala-lang.modules/scala-xml_2.12.svg?label=scala+2.12)](http://mvnrepository.com/artifact/org.scala-lang.modules/scala-xml_2.12) [![latest release for 2.13.0-M1](https://img.shields.io/maven-central/v/org.scala-lang.modules/scala-xml_2.13.0-M1.svg?label=scala+2.13.0-M1)](http://mvnrepository.com/artifact/org.scala-lang.modules/scala-xml_2.13.0-M1) [![Gitter](https://badges.gitter.im/Join+Chat.svg)](https://gitter.im/scala/scala-xml)

**Comment by ashawley** [comment]
Fixed with  f0c1e5f
